cirq/ops/fsim_gate.py

```diff
@@ -164,3 +165,252 @@ def __repr__(self) -> str:
 
     def _json_dict_(self) -> Dict[str, Any]:
         return protocols.obj_to_dict_helper(self, ['theta', 'phi'])
+
+
+@value.value_equality(approximate=True)
+class PhasedFSimGate(gate_features.TwoQubitGate,
+                     gate_features.InterchangeableQubitsGate):
```

Comment:
I just started to wonder about this: what does it mean that this gate implements InterchangeableQubitsGate? Does it mean that operation is equal after qubits are swapped? That's not true, right?

Reply:
Thanks for catching this! Sent #3524.